### PR TITLE
Trinity: multiple input and small cosmetic changes

### PR DESCRIPTION
**Pull request by abretaud**
Hi,
Here are some small changes to trinity: now it can have multiple input fastqs, and I fixed some option names too

### CODE REVIEW

**Comment by abretaud** [comment]
Is there someone available to review this? I need it in a somewhat urgent workflow...

File: tools/trinity/trinity.xml [hunk 1]
```diff
@@ -11,34 +11,37 @@
 
         ## Inputs.
         #if $inputs.paired_or_single == "paired":
-            --left $inputs.left_input --right $inputs.right_input
-            
-            #if $inputs.left_input.is_of_type('fasta'):
+
+            --left ${ ','.join(map( lambda x:'"%s"' % ( x ), $inputs.left_input )) }
```
Comment (nsoranzo) [inline tools/trinity/trinity.xml]: map( lambda x:'"%s"' % ( x ), $inputs.left_input )

can be simplified to:

['"%s"' % x for x in $inputs.left_input]

Same below.

**Comment by abretaud** [comment]
Thank you, it's done!

**Comment by bgruening** [comment]
Great! Thanks a lot!

**Comment by abretaud** [comment]
Sorry to bother you, is it possible to push this to MTS please?
Anthony (under users pressure :) )

**Comment by nsoranzo** [comment]
MTS seems to be down, @martenson @davebx can you help?

**Comment by abretaud** [comment]
MTS seems to be back (at least for me!)

**Comment by martenson** [comment]
Yes, it is constantly running out of space for bad reasons, we are trying to fix that. Sorry :/

**Comment by nsoranzo** [comment]
MTS updated

**Comment by abretaud** [comment]
Excellent, thank you very much!

**Pull request merged**